src/commands/lambda/README.md

```diff
@@ -145,6 +145,34 @@ Instead of supplying arguments, you can create a configuration file in your proj
     }
 }
 ```
+
+## Troubleshooting
+### `flare`
+To troubleshoot issues you may be encountering with Datadog monitoring on your Lambda functions, use `datadog-ci lambda flare`. This command sends relevant files to Datadog's support team for investigation.
+
+Note: This command works whether or not your Lambda functions were instrumented using `datadog-ci lambda instrument`.
+
+```bash
+# Collect and send files to Datadog support for a single function
+datadog-ci lambda flare -f <function-name> -r us-east-1 -c <case-id> -e <email-on-case-id>
+
+# Include recent CloudWatch logs
+datadog-ci lambda flare -f <function-name> -r us-east-1 -c <case-id> -e <email-on-case-id> --with-logs
+
+# Dry run -- collect files, but don't send them to Datadog support
+datadog-ci lambda flare -f <function-name> -r us-east-1 -c <case-id> -e <email-on-case-id> --dry
+```
+
+| Argument      | Shorthand | Description                                                                                                         | Default |
+|---------------|------|---------------------------------------------------------------------------------------------------------------------| --- |
+| `--function`  | `-f` | The ARN of the Lambda function to gather data for, or the name of the Lambda function (`--region` must be defined). | |
+| `--region`    | `-r` | The default region to use when `--function` is defined by name and not by ARN.                                      | |
+| `--case-id`   | `-c` | The Datadog case ID to send the files to.                                                                           | |
+| `--email`     | `-e` | The email associated with the specified case ID.                                                                    | |
+| `--with-logs` |      | Collect recent CloudWatch logs for the specified function.                                                          | |
+| `--dry`       | `-d` | Collect files without sending them to Datadog support.                                                              | `false` |
```

Comment:
```suggestion
| `--dry`       | `-d` | Preview collected data which would be sent to Datadog support.                                                              | `false` |
```
Or something around those lines, I'm not sure about `which would be sent`, maybe a better wording. I'd ask for Docs suggestion.